Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -585,6 +591,7 @@ void RemovePage(Page page)
 				_removeControllers = _removeControllers.Remove(target);
 				ViewControllers = _removeControllers;
 			}
+			target.Dispose();
```

Comment:
somewhere i saw it https://github.com/xamarin/Xamarin.Forms/pull/5695
This change looks good, but is not related to this PR.